vehicle/as/CMakeLists.txt

```diff
@@ -1,66 +1,26 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
 project(as)
 add_compile_options(-std=c++14)
```

Comment:
This is redundant with `set(CMAKE_CXX_STANDARD 14)` and can be removed

Reply:
done